src/modules/api-server/index.ts

```diff
@@ -13,12 +13,11 @@ import { ExternalClient, ExternalClientFactory } from './external-client';
 export type {
   AllEndpoints,
   HttpRequester,
-  GraphQlRequester,
   ExternalClient,
   ExternalClientFactory,
 };
 
-export { ApplicationError };
+export { GraphQlRequester, ApplicationError };
```

Comment:
Why was this moved?

Reply:
It's changed from not only being a type that is exported but also the class / constructor